templates/patterns/modal/pattern-modal.html.twig

```diff
@@ -1,31 +1,28 @@
 {#
 /**
  * @file
- * Modal component.
+ * Modal pattern.
  */
 #}
-
-{% set _modal_id = modal_id ? 'modal_' ~ modal_id : 'modal' %}
-
 {% if btn_label is not empty %}
```

Comment:
I don't remember this, so we are having a way to render button and modal together?